### PR TITLE
Create Contributing file

### LINKED ISSUE #4: Add CONTRIBUTING guidelines
We need to create `CONTRIBUTING.md`
You can refer the [GitHub docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) for this.

### PR DESCRIPTION
**Pull request by avneesh0612**
fixes #4

### CODE REVIEW

**Review by suitupstan — CHANGES_REQUESTED** [review]
I would like to have a bit more details, till we don't get any code/content, you can follow https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md

**Comment by avneesh0612** [comment]
Okay

**Comment by avneesh0612** [comment]
Made some changes :D

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,127 @@
...
+The process described here has several goals:
+
+- Maintain the project's quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible!
+- Enable a sustainable system for maintainers to review contributions
+
+Please follow all instructions in [the template](https://github.com/krishguptadev/open-source-best-practices/blob/main/.github/pull_request_template.md)
+
+## Style Guide for Git Commit Messages :memo:
```
Comment (suitupstan) [inline CONTRIBUTING.md]:
```suggestion
## Style Guide for Git Commit Messages 📝

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,127 @@
...
+git push origin your-branch-name
+```
+
+**8.** Create a new [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) from `your-branch-name`
+
+**9.** 🎉 Congratulations! You've made your first pull request! Now, you should just wait until the maintainers review your pull request.
+
+## 📈 Getting started
+
+- 😕 Not sure where to start? Join our community on [Discord](https://discord.gg/eZPwdM52vn)
```
Comment (suitupstan) [inline CONTRIBUTING.md]:
```suggestion
```

**Pull request merged**